src/interfaces/store-interfaces/state-interface.ts

```diff
@@ -45,5 +45,5 @@ export interface StateModelIF {
 
 export interface StateIF {
   stateModel: StateModelIF
-  resourceModel: ResourceIF
+  resourceModel: IncorporationResourceIF
```

Comment:
Should this be type `IncorporationResourceIF | DissolutionResourceIF` ?

Reply:
Yes, good catch!

Reply:
Finally a good catch! All my other comments were fluff 😛

Reply:
haha psh, hardly. This was a good reminded of some pr tech debt i had left behind!